lib/internal/main/print_help.js

```diff
@@ -152,8 +152,8 @@ function format(
     else
       text += StringPrototypeRepeat(' ', firstColumn - displayName.length);
 
-    text += indent(StringPrototypeTrimLeft(fold(displayHelpText, secondColumn),
-                                           firstColumn)) + '\n';
+    text += StringPrototypeTrimLeft(
+      indent(fold(displayHelpText, secondColumn), firstColumn)) + '\n';
```

Comment:
The left trim would always undo the effect of the `indent()` call here now, wouldn't it?